### ISSUE TITLE
Toolchains not working as expected on Windows

### ISSUE THREAD

**Issue by TheElan** [SinTh0r4s/ExampleMod1.7.10#33]
Try to find out what causes wired behaviour, on system where only jre was present and dev used a bundled version of 11th in IDEA there was a problem of gradle not downloading jdk and attempting to use jre.

Running `./gradlew build` should ensure proper java setup

Test cases to try out:
1. No java in the system - gradle should automatically install 8th when
2. Newer (11th) java in the system - gradle should automatically install 8th when
3. 8th jre in the system - gradle should install 8th jdk
3. Newer (11th) jre in the system - gradle should install 8th jdk

**Comment by TheElan** [comment]
We might want to consider rolling back to using if-throw exception on gradle evaluation if incorrect java is used (this means you need gradle to run on this java entirely, in comparison toolchain only forces it in some tasks)

**Comment by SinTh0r4s** [comment]
Yes. I think this is the way to go, untill all forge Gradle tasks can be converted to toolchains from Gradle 6
